src/styles/fonts.ts

```diff
@@ -1,26 +1,26 @@
 import { css } from 'styled-components'
 
-import AvertaRegular from '../assets/fonts/Averta-Regular.otf'
-import AvertaBold from '../assets/fonts/Averta-Bold.otf'
-import AvertaExtraBold from '../assets/fonts/Averta-Extrabold.otf'
+import InterRegular from '../assets/fonts/Inter-Regular.woff2'
+import InterBold from '../assets/fonts/Inter-Bold.woff2'
+import RobotoMono from '../assets/fonts/RobotoMono-Regular.woff2'
 
 const fontFace = css`
   @font-face {
-    font-family: 'Averta';
-    src: url(${AvertaRegular}) format('opentype');
-    font-weight: 500;
+    font-family: 'Inter';
+    src: url(${InterRegular}) format('opentype');
```

Comment:
for *.woff2 use `format('woff2')`
opentype is for *.otf

Reply:
Oops, forgot to update the format, my bad.

Reply:
Updated.